packages/client/tsconfig.eslint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "dnd/*",
+    "validation/*",
+    "utils/*",
+    "types/*",
+    "styles/*",
+    "stories/*",
+    "components/*",
+    "mutations/*",
+    "hooks/*"
+  ]
+}
```

Comment:
was getting tons of `The file does not match your project config` errors. this file fixes it.